03-attribute-operations.Rmd

```diff
@@ -2,78 +2,88 @@
 
 ## Prerequisites {-}
 
-- This chapter requires **tidyverse** and **sf**:
+- This chapter requires the packages **tidyverse** and **sf**:
 
 ```{r, message=FALSE}
 library(sf)
 library(tidyverse)
 ```
 
-- You must have loaded the `world` and `worldbank_df` data which are loaded automatically by the **spData** package:
+- We will also make use of the the `world` and `worldbank_df` data sets. Note that loading the **spData** package automatically attaches these data sets to your global environment:
 
 ```{r, results='hide'}
 library(spData)
 ```
 
 ## Introduction
 
-Attribute data is non-spatial information associated with geographic data.
-In the context of simple features, introduced in the previous chapter, this means a data frame with a column for each variable and one row per geographic feature stored in the `geom` list-column of `sf` objects.
+Attribute data is non-spatial information, e.g., the name of a bus station, associated with geographic data, e.g. the coordinate of this bus station.
```

Comment:
Not keen on using acronyms such as `i.e.` or `e.g.` mid-text, especially when it's surrounded by enclosing commas - does not flow great. I propose the line is changed to the following:

```
Attribute data is non-spatial information associated with geographic (geometry) data.
A bus station, for example, could be represented by a field containing it's name (attribute data), associated with its latitude and longitude position (geometry data).
```